pkg/kernel/patches-5.10.x/0014-Add-support-for-LEDs-on-Operational-Services-EM320-d.patch

```diff
@@ -0,0 +1,244 @@
+From 4dfe391597a1c40dfc00833922ee06bbc5f7dbed Mon Sep 17 00:00:00 2001
+From: Mikhail Malyshev <mikem@zededa.com>
+Date: Wed, 31 Aug 2022 15:21:26 +0200
+Subject: [PATCH] Add support for LEDs on Operational Services EM320 device
+
+Signed-off-by: Mikhail Malyshev <mikem@zededa.com>
+---
+ drivers/platform/x86/Kconfig    |  10 ++
+ drivers/platform/x86/Makefile   |   4 +
+ drivers/platform/x86/os-em32x.c | 189 ++++++++++++++++++++++++++++++++
+ 3 files changed, 203 insertions(+)
+ create mode 100644 drivers/platform/x86/os-em32x.c
+
+diff --git a/drivers/platform/x86/Kconfig b/drivers/platform/x86/Kconfig
+index a1858689d6e1..9db5862aa4b4 100644
+--- a/drivers/platform/x86/Kconfig
++++ b/drivers/platform/x86/Kconfig
+@@ -1419,6 +1419,16 @@ config INTEL_TELEMETRY
+ 	  directly via debugfs files. Various tools may use
+ 	  this interface for SoC state monitoring.
+ 
++config OS_EM32X
++	tristate "Operational Services  EM32x platform"
++	depends on X86_64
++	depends on I2C
++	depends on LEDS_CLASS
++	depends on LEDS_GPIO
++	depends on GPIO_PCA953X
++	help
++	  This driver add support for Operational Services EM32x platforms
++
+ endif # X86_PLATFORM_DEVICES
+ 
+ config PMC_ATOM
+diff --git a/drivers/platform/x86/Makefile b/drivers/platform/x86/Makefile
+index 5f823f7eff45..5782260e7916 100644
+--- a/drivers/platform/x86/Makefile
++++ b/drivers/platform/x86/Makefile
+@@ -149,3 +149,7 @@ obj-$(CONFIG_INTEL_TELEMETRY)		+= intel_telemetry_core.o \
+ 					   intel_telemetry_pltdrv.o \
+ 					   intel_telemetry_debugfs.o
+ obj-$(CONFIG_PMC_ATOM)			+= pmc_atom.o
++
++# OS VW boxes
```

Comment:
I think "OS EM32x platform" or "Operational Services EM32x platform" would be better for this comment...

Reply:
fixed